### PR TITLE
[DOCS] Using ES|QL in Kibana

### PR DESCRIPTION
**Pull request by abdonpijpelink**
Adds docs about using ES|QL in Kibana. [Preview](https://elasticsearch_101324.docs-preview.app.elstc.co/guide/en/elasticsearch/reference/master/esql-kibana.html).

### CODE REVIEW

File: docs/reference/esql/esql-kibana.asciidoc [hunk 1]
```diff
@@ -2,14 +2,240 @@
...
+----
+
+[discrete]
+[[esql-kibana-time-filter]]
+=== Time filtering
+
+To display data within a specified time range, use the
+{kibana-ref}/set-time-filter.html[time filter]. The time filter is only enabled
+when the indices you're querying contain a field called `@timestamp`. If your
+indices do not contain an `@timestamp` field, limit the time range using the
```
Comment (amyjtechwriter) [inline docs/reference/esql/esql-kibana.asciidoc:147]:
```suggestion
indices do not contain a `@timestamp` field, limit the time range using the
```

File: docs/reference/esql/esql-kibana.asciidoc [hunk 2]
```diff
@@ -2,14 +2,240 @@
...
+
+[source,esql]
+----
+FROM kibana_sample_data_logs
+| STATS total_bytes = SUM(bytes) BY geo.dest
+| SORT total_bytes DESC
+| LIMIT 3
+----
+
+changes the visualization into a bar chart showing the top 3 countries:
```
Comment (amyjtechwriter) [inline docs/reference/esql/esql-kibana.asciidoc:170]: I think you're correct not to capitalize this 'c' in 'change' here, but maybe we should get a second opinion.

**Review by amyjtechwriter — APPROVED** [review]
Looks great to me!

File: docs/reference/esql/esql-kibana.asciidoc (hunk 2 above)
Comment (abdonpijpelink) [reply docs/reference/esql/esql-kibana.asciidoc]: I wasn't sure about this one. It didn't look right, so I have changed it. Thanks!

File: docs/reference/esql/esql-kibana.asciidoc (hunk 1 above)
Comment (abdonpijpelink) [reply docs/reference/esql/esql-kibana.asciidoc]: It's pronounced "at timestamp", so I'll keep it.

**Review by stratoula — APPROVED** [review]
I couldn't write it better, this is awesome! LGTM!

**Pull request merged**